website/docs/artist_tools.md

```diff
@@ -177,6 +177,22 @@ Library loader is extended [loader](#loader) which allows to load published subs
 </div>
 </div>
 
+### Delivery loader ###
```

Comment:
We should call it `Delivery Action`, loader is confusing from user perspective

Reply:
Ok, renamed.